packages/hub/services/serializers/email-card-drop-request-serializer.ts

```diff
@@ -0,0 +1,29 @@
+import { JSONAPIDocument } from '../../utils/jsonapi-document';
+
+interface EmailCardDropRequestClaimStatus {
+  ownerAddress: string;
+  claimed: boolean;
+  timestamp: Date;
+}
+
+export default class EmailCardDropRequestSerializer {
+  serializeEmailCardDropRequestStatus(model: EmailCardDropRequestClaimStatus): JSONAPIDocument {
+    return {
+      data: {
+        type: 'email-card-drop-request-claim-status',
+        id: `${model.ownerAddress}-${model.timestamp.toISOString()}`,
+        attributes: {
+          'owner-address': model.ownerAddress,
+          claimed: model.claimed,
+          timestamp: model.timestamp.toISOString(),
+        },
+      },
+    } as JSONAPIDocument;
```

Comment:
Does it need to be casted if it's already defined on the return of the function ?

Reply:
No, I don't think it does - and local build works without too, so I'll remove this. Thanks!